Kubernetes/windows/debug/networkmonitor/networkhealth.ps1

```diff
@@ -986,8 +986,8 @@ class LoadBalancerStaleRemoteEndpoints : DiagnosticTest {
         }
         $stale_endpoints = [System.Collections.ArrayList] ($lb_endpoints | Select-Object -Unique)
         foreach($endpoint in $endpointsData) {
-            if($stale_endpoints.Contains($endpoint.Identifier)) {
-                $stale_endpoints.remove($endpoint.Identifier)
+            if($stale_endpoints -Contains $endpoint.Identifier) {
+                $stale_endpoints = $stale_endpoints -ne $endpoint.Identifier
```

Comment:
is -ne is a comparison operator? Or does it also modify stale_endpoints container?

Reply:
yes, and yes.